internal/ui/ui_js.go

```diff
@@ -702,6 +702,7 @@ func (u *UserInterface) setCanvasEventHandlers(v js.Value) {
 	}))
 }
 
+// appendDroppedFiles will only take the first file, all other files that was drop will get thrown
```

Comment:
Remove the comment. We don't know the case when multiple 'file' objects come yet.

Reply:
When multiple files are dropped onto the canvas, all of them will be available in the `items` property of the drop event. However, it's important to highlight that with the current implementation, `DroppedFiles` will only contain the first file from this selection. This means if you drop three files, only one will be processed and stored in DroppedFiles. To handle all dropped files, you'd need to modify the code to iterate through the items and process each file individually.

Reply:
> When multiple files are dropped onto the canvas, all of them will be available in the items property of the drop event. 

Have you confirmed this?

Reply:
As I said, the current implementation alredy works. I don't know when multiple 'file' objects come.

https://github.com/user-attachments/assets/365be2f9-c8f5-49ae-a8a4-6b1a4e7320c5